root/static/scripts/edit/URLCleanup.js

```diff
@@ -1460,7 +1463,7 @@ const CLEANUPS = {
           case LINK_TYPES.otherdatabases.work:
             return prefix == 'works/work';
           case LINK_TYPES.otherdatabases.artist:
-            return prefix !== 'works/work';	
+            return prefix !== 'works/work';
```

Comment:
Not sure how this happened (my IDE or me) but it seems it makes it more consistent so I didn't revert anyway.

Reply:
It was inconsistent, thus it is fine to remove it, but it should preferably be done in a separate commit. Anyway, https://github.com/metabrainz/musicbrainz-server/commit/d247b5c219ae231b9da2e24b4de3d70583601624#diff-87814c26df342be7e4c2a85531bbd47cL1463 has already been merged.